conf/app.routes

```diff
@@ -18,17 +18,17 @@ GET         /file-upload-failed                     controllers.FileUploadContro
 
 GET         /cannot-upload-another-file             controllers.FileUploadController.uploadInProgress
 
-GET         /member-name                            controllers.MemberNameController.get
-POST        /member-name                            controllers.MemberNameController.post
-GET         /member-name/back                       controllers.MemberNameController.back
+GET         /member-name                            controllers.MemberNameController.get(edit: Boolean ?= false)
```

Comment:
@joepowell96  @eulogiogutierrez having URL parameters doesnt seem right to me with the user able to change might have issues and also there would be accessibility testing and pen testing issues, but for the purpose of ticket we  will merge this but will raise a refactor story to improve this